website/vercel.json

```diff
@@ -865,6 +865,15 @@
     {
       "source": "/docs/plugins/overview",
       "destination": "https://hub.cloudquery.io"
+    },
+    {
+      "source": "/docs/core-concepts/policies",
```

Comment:
I think only the first redirect that matches take, so not sure we should have 2 redirects with the exact same source.